common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -219,6 +219,10 @@ def modulestore():
             from lms.djangoapps.ccx.modulestore import CCXModulestoreWrapper
             _MIXED_MODULESTORE = CCXModulestoreWrapper(_MIXED_MODULESTORE)
 
+        for wrapper_path in settings.MODULESTORE_WRAPPERS:
+            wrapper = load_function(wrapper_path)
+            _MIXED_MODULESTORE = wrapper(_MIXED_MODULESTORE)
```

Comment:
@jimabramson an invalid or broken wrapper won't prevent the app from starting, but will cause AttributeErrors to be raised when accessing any course content. I thought about wrapping this in an exception handler, but I'd prefer that broken wrappers not be ignored.